src/ClientData/ProcessData.cpp

```diff
@@ -66,15 +66,15 @@ void ProcessData::UpdateModuleInfos(absl::Span<const ModuleInfo> module_infos) {
 
   for (const auto& module_info : module_infos) {
     {
-      const auto [it, success] = module_memory_map_.try_emplace(
-          module_info.file_path(),
-          ModuleInMemory{module_info.address_start(), module_info.address_end(),
-                         module_info.file_path(), module_info.build_id()});
+      const auto [unused, success] = module_memory_map_.try_emplace(
```

Comment:
```suggestion
      const auto [unused_it, success] = module_memory_map_.try_emplace(
```
This is the format we had decided on.

Reply:
done.